app/src/main/java/com/acsbendi/requestinspectorwebview/RequestInspectorJavaScriptInterface.kt

```diff
@@ -106,6 +138,18 @@ internal class RequestInspectorJavaScriptInterface(webView: WebView) {
         return map
     }
 
+    private fun getFormParametersAsMap(formParameterJsonArray: JSONArray): Map<String, String> {
+        val map = HashMap<String, String>()
+        repeat(formParameterJsonArray.length()) { i ->
+            val formParameter = formParameterJsonArray.get(i) as JSONObject
+            val name = formParameter.getString("name")
+            val value = formParameter.getString("value")
+            map[name] = value
+        }
+        return map
```

Comment:
Could we simplify this using `formParameterJsonArray.map { ... }` rather than explicit iteration (`repeat`)?

Reply:
there is no map extension function for a JSONArray (

Reply:
Oh that's true, [it doesn't support iterators ](https://stackoverflow.com/questions/4784874/is-it-possible-to-iterate-through-jsonarray) :(